presets/base/linux.config

```diff
@@ -1,17 +1,22 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 5.11.21 Kernel Configuration
+# Linux/x86 5.16.4 Kernel Configuration
```

Comment:
should probably be two commits, with more info on config related base preset changes, but whatever :) lets say it was mostly enabling ACPI and olddefconfig...